optaweb-vehicle-routing-frontend/package-lock.json

```diff
@@ -5,16 +5,12 @@
   "dependencies": {
     "@babel/code-frame": {
       "version": "7.0.0",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
```

Comment:
@rsynek How about NO?

Reply:
Yes, this is one big no. And this change most likely comes from the locktt. Still, I don't understand how this happened: both locally and in my testing pipeline, there were no such changes in this file.

Locally, I managed to reproduce it just by running `npm run locktt`.